src/import-data/ImportDataDestination.ts

```diff
@@ -358,15 +363,20 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
               h(NewWorkspaceModal, {
                 requiredAuthDomain: requiredAuthorizationDomain,
                 cloudPlatform: getCloudPlatformRequiredForImport(importRequest),
```

Comment:
Screenshots of various options.

With and without "import may take time", no policies:
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/009f1f68-8ad6-4d7d-b208-83e6b24b3fc9)
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/b33d3d51-7748-47e8-b36f-c0ada8bd4ce5)

With and without "import may take time", no billing project selected:
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/44ca6914-fda7-4652-a976-e63120e1b5b9)

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/fb6f00f1-acdd-42f4-b5c6-be9e2793fe74)

Reply:
@cahrens Can this modal grow easily with more policies? For example, the Data Access Control policy currently exists, and I wonder if three listed policies would impact the design or would it fit pretty seamlessly?

Reply:
Yes, it will be able to display more policies… it just iterates over whatever policies are present.

Reply:
@nawatts here it is with all the policies.

<img width="571" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/00a6a5eb-a894-48e3-b73b-aea0826e7072">